tex/generic/pgf/frontendlayer/tikz/tikz.code.tex

```diff
@@ -2120,10 +2120,14 @@
   \else%
     \tikz@invoke@collected@onpath%
   \fi%
-  \afterassignment\tikz@handle\let\pgf@let@token=%
+  \tikz@scan@@next@command
 }%
+\def\tikz@scan@@next@command{%
```

Comment:
How about your original suggestion?
```suggestion
\def\tikz@handle@next{%
```
If would also be nice to write a short docstring.

Reply:
Original comment: Because it is used by `\tikz@scan@next@command`, how about keeping the current name (which makes it a simplified variant of `\tikz@scan@next@command`), and applying your `\tikz@handle@evenmore` with a docstring?

I forgot that the PR is based on branches hence new commits will automatically added. Sorry I just did what I prefer before receiving your reply.

Reply:
Don't worry.  You can push as many commits as you want.  If it becomes too messy you can always rebase.

Reply:
Sounds good to me.